src/components/About/About.jsx

```diff
@@ -1,41 +1,44 @@
- import React from "react";
+import React from "react";
+import { renderBody as render } from "SHARED/renderUtils.jsx";
+import { data } from "./";
+import { s } from "./";
 
 export default function About(){
 
-	const data = {
-		image: {
-			src: "path/to/whatever.jpg",
-			description: "A bunch of vikings screaming up (improve this later)."
-		},
-		title: "About",
-		subtitle: "The world is coming to an end.",
-		body: "cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-	};
-
 	const { 
-		image, 
+		image: heroImg, 
 		title, 
 		subtitle, 
-		body 
+		body
 	} = data;
 
 	const { 
 		src, 
 		description 
-	} = image;
+	} = heroImg;
+
+	const bodyContent = render(body); // create spacing at line-breaks
```

Comment:
so, what I usually do to solve this naming problem, is alias the data-object as `**Data` to preserve the best variable names for the render return, so in this case we'd have:

```
const {
    ...,
    body: bodyData
} = data;

const body = RENDER.body(bodyData);
```